bitcoin/src/blockdata/transaction.rs

```diff
@@ -1966,18 +1967,18 @@ mod tests {
 
 #[cfg(bench)]
 mod benches {
-    use hex::test_hex_unwrap as hex;
     use io::sink;
     use test::{black_box, Bencher};
 
     use super::*;
     use crate::consensus::{deserialize, Encodable};
+    use crate::hex::FromHex as _;
 
     const SOME_TX: &str = "0100000001a15d57094aa7a21a28cb20b59aab8fc7d1149a3bdbcddba9c622e4f5f6a99ece010000006c493046022100f93bb0e7d8db7bd46e40132d1f8242026e045f03a0efe71bbb8e3f475e970d790221009337cd7f1f929f00cc6ff01f03729b069a7c21b59b1736ddfee5db5946c5da8c0121033b9b137ee87d5a812d6f506efdd37f0affa7ffc310711c06c7f3e097c9447c52ffffffff0100e1f505000000001976a9140389035a9225b3839e2bbf32d826a1e222031fd888ac00000000";
 
     #[bench]
     pub fn bench_transaction_size(bh: &mut Bencher) {
-        let raw_tx = hex!(SOME_TX);
+        let raw_tx = Vec::from_hex(SOME_TX).unwrap();
```

Comment:
This also shouldn't need `Vec`, might need cast to `&[u8]` below.

Reply:
Added a patch up front that cleans up the bench code in `transaction`.